community_images/consul/bitnami/docker-compose.yml

```diff
@@ -0,0 +1,94 @@
+version: '2'
+
+services:
+  consul-node1:
+    image: ${CONSUL_IMAGE_REPOSITORY}:${CONSUL_IMAGE_TAG}
+    user: root
+    restart: always
+    environment:
+      - CONSUL_BOOTSTRAP_EXPECT=3
+      - CONSUL_CLIENT_LAN_ADDRESS=0.0.0.0
+      - CONSUL_DISABLE_KEYRING_FILE=true
+      - CONSUL_RETRY_JOIN_ADDRESS=consul-node1
+    networks:
+      - consul
+    ports:
+      - '0.0.0.0::8300'
+      - '0.0.0.0::8301'
+      - '0.0.0.0::8301/udp'
+      - '0.0.0.0::8500'
+      - '0.0.0.0::8600'
+      - '0.0.0.0::8600/udp'
+    cap_add:
+      - SYS_PTRACE
+    volumes:
+      - 'consul-node1_data:/bitnami'
+      - ./scripts/server_coverage_script.sh:/opt/bitnami/scripts/coverage_script.sh
+      - ./configs/sample_service.json:/consul.d/sample_service.json
+
+  consul-node2:
+    image: ${CONSUL_IMAGE_REPOSITORY}:${CONSUL_IMAGE_TAG}
+    user: root
+    restart: always
+    environment:
+      - CONSUL_BOOTSTRAP_EXPECT=3
+      - CONSUL_CLIENT_LAN_ADDRESS=0.0.0.0
+      - CONSUL_DISABLE_KEYRING_FILE=true
+      - CONSUL_RETRY_JOIN_ADDRESS=consul-node1
+      - CONSUL_ENABLE_UI=true
+    networks:
+      - consul
+    cap_add:
+      - SYS_PTRACE
+    depends_on:
+      - consul-node1
+    volumes:
+      - 'consul-node2_data:/bitnami'
+
+  consul-node3:
+    image: bitnami/consul
```

Comment:
fix this image name

Reply:
It's a client/original container. I'm using apt-get here to install dnsutils package. The purpose of this container is to Query a service using dns api over the consul network. (I can not use the dig command in the stubbed images) 
However, I did find one addition issue here. The query would return no positive result since I'm registering as well as deregistering the service in the consul-node1's coverage script.

Reply:
got it, we can leave it as it is in that case.